OCPP-J/src/main/java/eu/chargetime/ocpp/WebSocketListener.java

```diff
@@ -145,14 +147,9 @@ public void onError(WebSocket webSocket, Exception ex) {
                 (webSocket != null)
                     ? webSocket.getResourceDescriptor()
                     : "not defined (webSocket is null)";
-
-            if (ex instanceof ConnectException) {
-              logger.error(
-                  "On error (resource descriptor: " + resourceDescriptor + ") triggered caused by:",
-                  ex);
-            } else {
-              logger.error(
-                  "On error (resource descriptor: " + resourceDescriptor + ") triggered:", ex);
```

Comment:
Exception is already logged to logger.error by the WebSocket library before this method, `onError`, is called.